deployment/docker-compose.yml

```diff
@@ -61,11 +60,23 @@ uwsgi:
     - ./media:/home/web/media:rw
     - ./reports:/home/web/reports
     - ./logs:/var/log/
+  user: root
+
+uwsgi-ephemeral:
+  extends: uwsgi-base
+  container_name: mezzanine-uwsgi-ephemeral
```

Comment:
So we don't expect it to be running, so exec'ing we will do into mezzanine-uwsgi. 
migration: docker-compose -p $(PROJECT_ID) run uwsgi-ephemeral python manage.py migrate 
shell: docker-compose -p $(PROJECT_ID) run uwsgi-ephemeral python manage.py shell